modules/ixBidAdapter.js

```diff
@@ -481,6 +481,31 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
   return requests;
 }
 
+/**
+ * Return an object of user IDs stored by Prebid User ID module
+ *
+ * @returns {Object} ID providers and whether they are present
+ */
+function _getUserIds(bidRequest) {
+  const userIds = bidRequest.userId || {};
+
+  return {
+    'britepoolid': utils.deepAccess(userIds, 'britepoolid') ? 1 : 0,
+    'id5id': utils.deepAccess(userIds, 'id5id') ? 1 : 0,
```

Comment:
specifically for ID5, this will provide back an object that includes the ID5 ID and our extension (which includes the `linkType` field that DSPs are expecting).